Dockerfile

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
😢 was going to suggest `Dockerfile` -> `Containerfile` but seems like `Containerfile` is just as podman specific as `Dockerfile` is docker specific and they [don't want](https://github.com/docker/cli/issues/3251#issuecomment-1063338859) to support it